### PR TITLE
(AUTOMATIC) opensource update

### PR DESCRIPTION
**Pull request by sebsg**
Makes the address normalizer's normalize function check all languages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,37 +0,0 @@
-# Copyright 2017 Google Inc.
```
Comment (rsolomakhin) [inline .travis.yml]: Can we not delete this plz?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # ![](https://github.com/googlei18n/libaddressinput/wiki/libaddressinput-icon-70x55.png) libaddressinput
 
-[![Build Status](https://travis-ci.org/googlei18n/libaddressinput.svg?branch=master)](https://travis-ci.org/googlei18n/libaddressinput)
```
Comment (rsolomakhin) [inline README.md]: And let's keep this as well.

File: cpp/src/address_normalizer.cc [hunk 3]
```diff
@@ -60,23 +68,28 @@ void AddressNormalizer::Normalize(AddressData* address) const {
...
-      if (matches_latin_name || matches_local_name_id) {
-        address->SetFieldValue(
-            field, matches_latin_name ? rule->GetLatinName() : *key_it);
-        no_match_found_yet = false;
-        parent_key.FromLookupKey(parent_key, *key_it);
-        parent_rule = supplier_->GetRule(parent_key);
-        assert(parent_rule != NULL);
+      if (!no_match_found_yet)
         break;
+      for (std::string language : languages) {
```
Comment (rsolomakhin) [inline cpp/src/address_normalizer.cc]: This makes string copies. Needs to be "const std::string& language".

File: .travis.yml (hunk 1 above)
Comment (sebsg) [inline .travis.yml]: Done.

File: README.md (hunk 2 above)
Comment (sebsg) [inline README.md]: Done

File: cpp/src/address_normalizer.cc (hunk 3 above)
Comment (sebsg) [inline cpp/src/address_normalizer.cc]: Done

**Comment by sebsg** [comment]
I've also modified the source so that the next automatic roll doesn't delete these changes again.

**Pull request merged**